src/psycop_model_evaluation/binary/time/timedelta_plots.py

```diff
@@ -157,6 +157,7 @@ def plot_time_from_first_positive_to_event(
     eval_dataset: EvalDataset,
     min_n_in_bin: int = 0,
     bins: Sequence[float] = tuple(range(0, 36, 1)),  # noqa
+    bin_unit: Literal["h", "D", "M", "Q", "Y"] = "M",
```

Comment:
We probably want to extract `Literal["h", "D", "M", "Q", "Y"]` into its own variable, e.g.:

`bin_unit_literals = Literal["h", "D", "M", "Q", "Y"]`

And then use it everywhere, so we don't need to keep them in sync:

```suggestion
    bin_unit: bin_unit_literals = "M",
```